src/vivarium_testing_utils/automated_validation/interface.py

```diff
@@ -197,7 +202,7 @@ def metadata(self, comparison_key: str) -> pd.DataFrame:
     def get_frame(
         self,
         comparison_key: str,
-        stratifications: Collection[str] = (),
+        stratifications: Collection[str] | None = None,
```

Comment:
Do we like this approach having `None` mean the opposite of `[]`? We have something similar when requesting columns for population views, and I think it's confusing. Would it be clearer just to split it into two parameters `is_stratified` and `stratifications` and not allowing `None` here?

Reply:
Record keeping: We decided to move to use the literal "all" instead of none.